migration-util/src/main/java/edu/unc/lib/dcr/migration/premis/Premis2Constants.java

```diff
@@ -33,6 +33,7 @@ private Premis2Constants() {
     public static final String DELETION_TYPE = "http://id.loc.gov/vocabulary/preservationEvents/deletion";
     public static final String FIXITY_CHECK_TYPE = "http://id.loc.gov/vocabulary/preservationEvents/fixityCheck";
     public static final String CREATION_TYPE = "http://id.loc.gov/vocabulary/preservationEvents/creation";
+    public static final String ACCESSION_TYPE = "http://id.loc.gov/vocabulary/preservationEvents/accession";
```

Comment:
Based on the previous comments, we don't have accession events in boxc3 so you shouldn't need this